packages/terminal/src/browser/base/terminal-widget.ts

```diff
@@ -19,7 +19,7 @@ import { BaseWidget } from '@theia/core/lib/browser';
 import { CommandLineOptions } from '@theia/process/lib/common/shell-command-builder';
 import { TerminalSearchWidget } from '../search/terminal-search-widget';
 import { TerminalProcessInfo } from '../../common/base-terminal-protocol';
-
+import { Uri } from '@theia/plugin';
```

Comment:
`@theia/terminal` does not want to depend on `@theia/plugin`. Please use the Theia `URI` from `@theia/core/lib/common/uri` in `@theia/terminal` and make the corresponding `Uri` to `URI` mapping if needed. Please change.

Reply:
@kittaakos I did a change regarding the URI. please review, thanks!